it/lessons/basics/collections.md

```diff
@@ -83,7 +83,7 @@ iex> t
 
 ## Tuple
 
-Le tuple sono simili alle liste, ma sono conservate in memoria in modo adiancente. Ciò permette di accedere agli elementi in modo rapido, but rende le modifiche più dispendiose; la nuova tupla deve essere interamente copiata in memoria. Le tuple sono rappresentate con le parentesi graffe:
+Le tuple sono simili alle liste, ma sono conservate in memoria in modo adiancente. Ciò permette di accedere agli elementi in modo rapido, ma rende le modifiche più dispendiose perchè viene generata una nuova nuova tupla deve essere interamente copiata in memoria. Le tuple sono rappresentate con le parentesi graffe:
```

Comment:
"perchè viene generata una nuova nuova tupla deve essere interamente copiata in memoria" => "perché quando viene generata una nuova nuova tupla, questa deve essere interamente copiata e spostata in una nuova locazione di memoria libera e contigua abbastanza grande da contenerla"